mkosi/distributions/gentoo.py

```diff
@@ -105,39 +119,41 @@ def install(cls, state: MkosiState) -> None:
                 die("profile names changed upstream?")
 
         stage3_tar_path = state.cache_dir / stage3_tar
-        cls.stage3_cache = stage3_tar_path.with_name(stage3_tar.name).with_suffix(".tmp")
 
         stage3_url_path = urllib.parse.urljoin(
             state.config.mirror, f"releases/{arch}/autobuilds/{stage3_tar}",
         )
-        if not stage3_tar_path.is_file():
-            log_step(f"Fetching {stage3_url_path}")
-            stage3_tar_path.parent.mkdir(parents=True, exist_ok=True)
-            urllib.request.urlretrieve(stage3_url_path, stage3_tar_path)
 
+        cls.stage3_cache = state.cache_dir.joinpath("stage3")
```

Comment:
Because we hardcode it now, there's no need for cls.stage3_cache anymore, please drop the class variable

Reply:
ok!